### PR TITLE
[Bug][API] list paging missing totalpage

### LINKED ISSUE #15606: [Bug] Version 3.2.1 [Instance list paging is incorrect, the monitored disk capacity is displayed as 0]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

刚升级到3.2.1，发现实例列表分页失败，还有监控的磁盘容量显示错误 全是0 
bug1
bug2
bug3

### What you expected to happen

解决实例列表无法分页问题

### How to reproduce

如上图，3.2.1版本

### Anything else

_No response_

### Version

3.2.x

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Gallardot**
## Purpose of the pull request

#15181 deleted some code by mistake

- close #15606 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by SbloodyS** [comment]
https://github.com/apache/dolphinscheduler/pull/15148 is closed. Which deleted some code by mistake? @Gallardot

**Comment by SbloodyS** [comment]
Did you mean #15181?

**Review by SbloodyS — APPROVED** [review]
LGTM.

**Comment by SbloodyS** [comment]
cc @ruanwenjun

**Comment by Gallardot** [comment]
> Did you mean #15181?

Yes, #15181

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java [hunk 1]
```diff
@@ -75,4 +77,15 @@ public static <T> PageInfo<T> of(IPage<T> iPage) {
     public static <T> PageInfo<T> of(Integer currentPage, Integer pageSize) {
         return new PageInfo<>(currentPage, pageSize);
     }
+
+    public Integer getTotalPage() {
```
Comment (DaZuiZui) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java:81]: This seems to achieve the same function, and is simpler
this.totalPage = Math.ceil(this.total / this.pageSize);

**Review by DaZuiZui — COMMENTED** [review]
This seems to achieve the same function, and is simpler
this.totalPage = Math.ceil(this.total / this.pageSize);

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java (hunk 1 above)
Comment (Gallardot) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java:81]: @DaZuiZui 
The original code is from a previously deleted section [#15181](https://github.com/apache/dolphinscheduler/pull/15181/files#diff-c367978011e1fa572823ec7cec246f9a21ad88699ef9c764ee39e9c96d23914b). But your suggestions are better, there is no reason not to optimize, you are welcome to create a PR❤️.

Comment (SbloodyS) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java:81]: > @DaZuiZui The original code is from a previously deleted section [#15181](https://github.com/apache/dolphinscheduler/pull/15181/files#diff-c367978011e1fa572823ec7cec246f9a21ad88699ef9c764ee39e9c96d23914b). But your suggestions are better, there is no reason not to optimize, you are welcome to create a PR❤️.

+1

Comment (DaZuiZui) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java:81]: I have submitted PR
https://github.com/apache/dolphinscheduler/pull/15620

**Comment by ruanwenjun** [comment]
Thanks for fixing this Bug?

**Comment by niyanchun** [comment]
our production environment meet this problem, When 3.2.2 release？

**Pull request merged**